UI/src/elements/lsmb-text.js

```diff
@@ -0,0 +1,78 @@
+
+const registry = require("dijit/registry");
+const dojoTextBox = require("dijit/form/TextBox");
+
+class LsmbText extends HTMLElement {
+    dojoWidget = null;
+
+    constructor() {
+        super();
+    }
+    adoptedCallback() {
+        if (this.dojoWidget
+            && this.dojoWidget.resize) {
+            this.dojoWidget.resize();
+        }
+    }
+    connectedCallback() {
+        if (this.dojoWidget) {
+            if (registry.byId(this.dojoWidget.id) !== this.dojoWidget) {
+                registry.add(this.dojoWidget);
```

Comment:
This code exists to compensate for the behaviour of `lsmb/layout/TableLayout` which moves widgets around in the dom and (re)registers them. However, this workaround to prevent double registration actually does not solve the problem. We'll need to do away with the TableLayout's behaviour as it is defined today.